packages/alchemy/src/schema.ts

```diff
@@ -0,0 +1,60 @@
+import { SmartAccountProviderConfigSchema } from "@alchemy/aa-core";
+import z from "zod";
+
+export const ConnectionConfigSchema = z
+  .object({
+    rpcUrl: z.never().optional(),
+    apiKey: z.string(),
+    jwt: z.never().optional(),
+  })
+  .or(
+    z.object({
+      rpcUrl: z.never().optional(),
+      apiKey: z.never().optional(),
+      jwt: z.string(),
+    })
+  )
+  .or(
+    z.object({
+      rpcUrl: z.string(),
+      apiKey: z.never().optional(),
+      jwt: z.never().optional(),
+    })
+  )
+  .or(
+    z.object({
+      rpcUrl: z.string(),
+      apiKey: z.never().optional(),
+      jwt: z.string(),
+    })
+  );
+
+export const FeeOptsSchema = z.object({
```

Comment:
can we define the defaults on these? preverification we probs cant, but the rest we should use the `.default` part of zod as well